src/stories/components/JsPsych/JsPsych.js

```diff
@@ -81,44 +80,50 @@ export const JsPsych = () => {
         };
         timeline.push(instructions);
 
+        // Preload images for experiment
+        var preload = {
```

Comment:
Preload plug in works amazing!! This is fabulous that we are able to render both images at the same time now.